util/util.py

```diff
@@ -456,7 +457,7 @@ def as_UUID( val, canbenone=True ):
 
 
 def as_datetime( string ):
-    """Convert a string to datetime.date with some error checking, allowing a null op.
+    r"""Convert a string to datetime.date with some error checking, allowing a null op.
```

Comment:
same question on if the r-string is needed?

Reply:
This one *is* needed because of the backslashes further down in the docstring.